### ISSUE TITLE
does not work

### ISSUE THREAD

**Issue by mimikun** [Omochice/dps-translate-vim#47]
I'm using this plugin in "neovim HEAD" and now I'm getting an error.

```txt
[denops] (translate) SyntaxError: The requested module 'https://deno.land/x/unknownutil@v3.6.0/mod.ts' does not provide
an export named 'assertString' at file:///home/mimikun/.local/share/nvim/lazy/dps-translate-vim/denops/translate/deps.ts
:7:3
[denops]     at async main (file:///home/mimikun/.local/share/nvim/lazy/denops.vim/denops/@denops-private/worker/script.
ts:71:17) {
[denops]   code: "ERR_MODULE_NOT_FOUND"
[denops] }
```

**Comment by mimikun** [comment]
I solved it by forking this repository and making the following changes:

```diff
diff --git a/denops/translate/deps.ts b/denops/translate/deps.ts
index 4bd06a2..574c43d 100644
--- a/denops/translate/deps.ts
+++ b/denops/translate/deps.ts
@@ -10,5 +10,5 @@ export {
   ensureString,
   isNumber,
   isString,
-} from "https://deno.land/x/unknownutil@v3.6.0/mod.ts";
+} from "https://deno.land/x/unknownutil@v2.1.1/mod.ts";
 export { openPopup } from "https://pax.deno.dev/Omochice/dps-popup-toy/mod.ts";
```

**Comment by Omochice** [comment]
oops, `unknownutil` v3 contains breaking changes...

I'll migrate it.

**Comment by Omochice** [comment]
I fix inyerfaces of unknownutil on 97b5ca27e5d0f34a86015f35bb1abb42468ef0a7 .

But some dependencies are outdated.
May need to re-cache deno module.

**Comment by mimikun** [comment]
Thanks!
but An error occurred and the translation result is not displayed.

<img width="612" alt="image" src="https://github.com/Omochice/dps-translate-vim/assets/13450321/8e833602-b0f8-4a84-80d6-87ae50a5cced">

```txt
[denops] at WritableStreamDefaultController.writeAlgorithm (ext:deno_web/06_streams.js:3773:14)
```

**Comment by Omochice** [comment]
Sorry, It looks the way is invalid to receive arguments...

I fixed it on 21e4a901 and checked with below version:

```
NVIM v0.10.0-dev-92760a7f4
Build type: Release
LuaJIT 2.1.0-beta3
Run "nvim -V1 -v" for more info
```

```
VIM - Vi IMproved 9.0 (2022 Jun 28, compiled Sep 02 2023 19:52:05)
Included patches: 1-1850
```

**Comment by mimikun** [comment]
It's solved.
Thanks!
